src/components/history-page/history-toolbar/history-view-mode-toggle-button.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import React, { useCallback } from 'react'
+import { ToggleButton, ToggleButtonGroup } from 'react-bootstrap'
+import { cypressId } from '../../../utils/cypress-attribute'
+import { ViewStateEnum } from './history-toolbar'
+import { useTranslation } from 'react-i18next'
+import { ForkAwesomeIcon } from '../../common/fork-awesome/fork-awesome-icon'
+import { useHistoryToolbarState } from './toolbar-context/use-history-toolbar-state'
+
+/**
+ * Toggles the view mode of the history entries between list and card view.
+ */
+export const HistoryViewModeToggleButton: React.FC = () => {
+  const { t } = useTranslation()
+  const [historyToolbarState, setHistoryToolbarState] = useHistoryToolbarState()
+
+  const onViewStateChange = useCallback(
+    (newViewState: ViewStateEnum) => {
+      setHistoryToolbarState((state) => ({
+        ...state,
+        viewState: newViewState
+      }))
+    },
+    [setHistoryToolbarState]
+  )
+
+  return (
+    <ToggleButtonGroup
+      type='radio'
+      name='options'
+      dir='ltr'
```

Comment:
Didn't we get the direction by our i18n framework already and could provide that here?

Reply:
This is how it was before and I don't want to change to many behaviours in this branch as it is only a refactoring.

Reply:
Okay new issue then.
https://github.com/hedgedoc/react-client/issues/1725